doc/dev_guide.rst

```diff
@@ -266,15 +266,30 @@ specification <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT
 .rst.txt>`_, as shown in `this example
 <https://github.com/numpy/numpy/blob/master/doc/example.py>`_.
 
-User-guide Documentation -- A description of the functionality of the code and
+User-guide documentation -- A description of the functionality of the code and
 how to use it with examples and links to the relevant code.
 
-Build the documentation -- To check the output of what you wrote, you can build
+Build user-guide documentation
+^^^^^^^^^^^^^^^^^^^
+
+To check the output of what you wrote, you can build
```

Comment:
What about replacing all this with:

```bash

pip install hyperspy[doc]
```

The advantage is that we only have to update the dependencies in setup.py, decreasing the risk of getting the docs out-of-sync.

Reply:
Done and rename `docs` to `build-doc` to avoid a confusion: user may think that it will install the doc and the doc build tools.